invokeai/app/invocations/generate.py

```diff
@@ -12,9 +12,10 @@
 from ..services.invocation_services import InvocationServices
 from .baseinvocation import BaseInvocation, InvocationContext
 from .image import ImageField, ImageOutput
+from ...backend.generator import Txt2Img, Img2Img, Inpaint, InvokeAIGenerator
 
 SAMPLER_NAME_VALUES = Literal[
-    "ddim", "plms", "k_lms", "k_dpm_2", "k_dpm_2_a", "k_euler", "k_euler_a", "k_heun"
+    tuple(InvokeAIGenerator.schedulers())
 ]
```

Comment:
I think we need/use an enum for the node anyway (which needs to be updated).

Reply:
I don't remember why I used a Literal here instead of an Enum. I either used an Enum here originally and changed it because it didn't work, or I used it elsewhere and never got around to updating this. I suspect the former, especially as this works at the moment.

Reply:
I'm more familiar with Enums than with Literal types, so I just took what @Kyle0654 had done and refactored so that the list of supported schedulers was only defined in one place. I'd be very happy to convert into Enums if it doesn't break anything.